docs/api/schedulers.md

```
@@ -0,0 +1,111 @@
+
+Public API:
+
+- `using context_type`
+- `post(sched_handle<context_type>) -> void`
+
+The pool is non-copyable and non-movable. Destruction requests stop on all
+workers and joins them through `std::jthread`.
+
+!!! note
+
+    `basic_busy_pool` currently busy-waits for work. It is useful for benchmark
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)